docs/reference/configuration/uds-monitoring-metrics.md

```diff
@@ -58,21 +46,72 @@ spec:
-2. We would need an "alert" mechanism (watch) to notify us when the service(s) are updated, to roll the corresponding updates to network policies and service monitors. While this is doable it feels like unnecessary complexity compared to other options.
+Grafana supports creation of folders for dashboards to provide better organization. UDS Core does not utilize folders by default but the sidecar supports simple values configuration to dynamically create and populate folders. The example overrides below show how to set this up and place the UDS Core default dashboards into a uds-core folder:
 
-### Generation of service + monitor
+```yaml
+  - name: core
+    repository: ghcr.io/defenseunicorns/packages/uds/core
+    ref: x.x.x
+    overrides:
+      grafana:
+        grafana:
+          values:
+            # This value allows us to specify a grafana_folder annotation to indicate the file folder to place a given dashboard into
+            - path: sidecar.dashboards.folderAnnotation
+              value: grafana_folder
+
+            # This value configures the sidecar to build out folders based upon where dashboard files are
+            - path: sidecar.dashboards.provider.foldersFromFilesStructure
+              value: true
+      kube-prometheus-stack:
+        kube-prometheus-stack:
+          values:
+            # This value adds an annotation to the defaults dashboards to specify that they should be grouped under a `uds-core` folder
+            - path: grafana.sidecar.dashboards.annotations
+              value:
+                grafana_folder: "uds-core"
+      loki:
+        uds-loki-config:
+          values:
+            # This value adds an annotation to the loki dashboards to specify that they should be grouped under a `uds-core` folder
+            - path: dashboardAnnotations
+              value:
+                grafana_folder: "uds-core"
+```
 
-Another alternative approach would be to use a pod selector and port only. We would then generate both a service and servicemonitor, giving us full control of the port names and selectors. This seems like a viable path, but does add an extra resource for us to generate and manage. There could be unknown side effects of generating services that could clash with other services (particularly with istio endpoints). This would otherwise be a relative straightforward approach and is worth evaluating again if we want to simplify the spec later on.
+:::note
+If using this configuration, any dashboards without a `grafana_folder` annotation will still be loaded in Grafana, but will not be grouped (they will appear at the top level outside of any folders). Also note that new dashboards in UDS Core may also need to be overridden to add the folder annotation, this example represents the current set of dashboards deployed by default.
```

Comment:
Maybe an explicit call out that dashboards that are managed out of UDS Core can also be grouped into separate folders with the dashboard_folder annotation. Update the example above?

Reply:
Updated, great callout.